src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -68,9 +68,11 @@ def validate_location(cmd, location):
             break
 
 
-def get_chart_path(registry_path, kube_config, kube_context):
+def get_chart_path(registry_path, kube_config, kube_context, container_registry_username="", container_registry_password=""):
+
     # Pulling helm chart from registry
     os.environ['HELM_EXPERIMENTAL_OCI'] = '1'
+    helm_login(registry_path, container_registry_username, container_registry_password)
```

Comment:
Looks like we are doing this step for non-pcr scenario as well, is it really required? I believe we could skip it

Reply:
We are not doing this for non PCR as a check is present inside the helm_login function to have better readability I have now moved the check outside